maintainers/maintainer-list.nix

```diff
@@ -2975,6 +2975,12 @@
     githubId = 8404455;
     name = "Diego Lelis";
   };
+  DieracDelta = {
```

Comment:
Put your credentials in a separate commit, with a message `maintainers: add DieracDelta`.

If you need some advice on how to do it, read this tutorials:

https://stackoverflow.com/questions/6217156/break-a-previous-commit-into-multiple-commits
https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History